### ISSUE TITLE
Allow fields in protocol grid view that are not applicable to all steps (Trac #69)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#73]
Currently, there is no mechanism to mark an entry in the protocol grid view as 'empty' or not applicable.  This functionality is required to enable settings conditional on other step field values (e.g., feedback options depend on mode, etc.).

Migrated from http://microfluidics.utoronto.ca/ticket/69
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "Currently, there is no mechanism to mark an entry in the protocol grid view as 'empty' or not applicable.  This functionality is required to enable settings conditional on other step field values (e.g., feedback options depend on mode, etc.).", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Allow fields in protocol grid view that are not applicable to all steps", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-03-14T15:53:36", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120315T05:15:05: ryan changed status from "new" to "assigned"

**Comment by cfobel** [comment]
Trac update at 20120329T00:23:05: 
* christian@fobel.net commented:

> commit c64aab41a39be8861c3932700d5cb3785113c08f
> Author: Christian Fobel <christian@fobel.net>
> Date:   Wed Mar 28 20:16:46 2012 -0400
> 
>     [SUBMODULE] Add support for mappers in protocol...
>     
>     [SUBMODULE] Add support for pygtkhelpers PropertyMapper lists as a Fields
>     item attribute.  Using this, it is possible to change the state of a cell in
>     the protocol grid view based on program state.  For example, as of this commit,
>     many of the feedback fields from the DMF control board plugin are disabled when
>     feedback is disabled for the corresponding step.
>     Fixes #73.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "assigned" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
